### PR TITLE
[FLINK-11003][docs] Fix mistake in java_lambda doc

### PR DESCRIPTION
**Pull request by KarmaGYZ**
## What is the purpose of the change

This pull request fix a mistake code in the Java Lambda Expressions doc

## Brief change log

- Change the generic type in MapFunction of MyTuple2Mapper class

## Verifying this change

This change is a docs cleanup.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

File: docs/dev/java_lambdas.md [hunk 1]
```diff
@@ -135,4 +135,4 @@ public static class DoubleTuple extends Tuple2<Integer, Integer> {
         this.f1 = f1;
     }
 }
-{% endhighlight %}
\ No newline at end of file
+{% endhighlight %}
```
Comment (zentol) [inline docs/dev/java_lambdas.md]: could you revert this change? It's probably just an automatically-added line-ending.

Comment (KarmaGYZ) [reply docs/dev/java_lambdas.md]: Done. Thanks for the review!
